pages/k8s/1.16/components.md

```diff
@@ -0,0 +1,329 @@
+---
+wrapper_template: kubernetes/docs/base_docs.html
+markdown_includes:
+  nav: kubernetes/docs/shared/_side-navigation.md
+context:
+  title: Components of Charmed Kubernetes 1.16
+  description: Detailed description of Charmed Kubernetes release
+keywords: component, charms, versions, release
+tags:
+    - reference
+sidebar: k8smain-sidebar
+permalink: 1.16/components.html
+layout:
+    - base
+    - ubuntu-com
+toc: false
+bundle_revision: '316'
+bundle_release: '1.16'
+---
+
+<!-- THIS PAGE IS AUTOGENERATED -->
+<!-- If you find an error in this page, it is likely to be in the original
+source of the information - please file a bug rather than editing the text -->
+
+This page details the charms, snaps, images and other components
+which go to make up the 1.16 release of Charmed Kubernetes.
+
+Other information about this release can be found on the following pages:
+
+<a class='p-button--brand' href='/kubernetes/docs'> Documentation </a>
+<a class='p-button--brand' href='/kubernetes/docs/1.16/release-notes'>Release notes </a>
+<a class='p-button--brand' href='/kubernetes/docs/1.16/upgrading'>Upgrade notes </a>
+<a class='p-button--brand' href='https://bugs.launchpad.net/charmed-kubernetes'>Bugs </a>
+<a class='p-button--brand' href='https://github.com/charmed-kubernetes/bundle'>Source </a>
+<a class='p-button--brand' href='https://launchpad.net/charmed-kubernetes/+milestone/1.16'>Milestone </a>
+
+## What's new
```

Comment:
On the one hand I like having everything in one place, but on the other hand, I feel like embedding the release notes here make the page feel really busy. And then if you click the Release Notes button above, you realize you're looking at the same content that you already saw here. I wonder if it would be better to call attention to the release notes here, but not embed them directly. e.g.:

## What's new
For a list of new features, changes, deprecations, and bug fixes in this release, please see the <a class='p-button--brand' href='/kubernetes/docs/1.16/release-notes'>Release notes </a>.

(or something like that)